pyaedt/application/Analysis.py

```diff
@@ -26,15 +26,16 @@
 
 
 class Analysis(Design, object):
-    """Analysis class.
-    
-    This is the main class, serving as a container for all common functions.
+    """Contains all common analysis functions.
     
     It is automatically initialized by a call from an application, such as HFSS or Q3D. 
     See the application function for its parameter descriptions.
 
     Parameters
     ----------
+    Design :
+    
+    object :
```

Comment:
```suggestion
    application : str
        Application that is to initialize the call.
    projectname : str
        Name of the project to select or the full path to the project
        or AEDTZ archive to open.
    designname : str
        Name of the design to select.
    solution_type : str
        Solution type to apply to the design.
    setup_name : str
        Name of the setup to use as the nominal.
    specified_version: str
        Version of AEDT  to use.
    NG : bool
        Whether to run AEDT in the non-graphical mode. 
    AlwaysNew : bool
        Whether to launch an instance of AEDT in a new thread, even if
        another instance of the ``specified_version`` is active on the
        machine.
    release_on_exit : bool
        Whether to release  AEDT on exit.
    student_version : bool
        Whether to enable the student version of AEDT. 
```
The `Design` and `object` shown in the Analysis class as `Analysis(Design, object)` mean that the class inherits from the `Design` and `object` classes. For the class parameters, the parameters for the `__init__` method is what matters.

Reply:
@jleonatti Thanks, Jess. I didn't know this!